README.md

```diff
@@ -42,7 +42,24 @@ node            |  |                               |  |
     ```shell
     make install
     ```
+## Docker Installation
 
+1. Prerequisites
+- Ensure you have Docker version **20.10** or higher installed. You can download it from [Docker's official website](https://www.docker.com/get-started).
+- Allocate at least **2 CPU cores** and **4 GB of RAM** to Docker for optimal performance. This can be configured in Docker Desktop settings.
```

Comment:
IMO we can assume that the user has done this already b/c it's part of Docker install steps

```suggestion
```